notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -1174,9 +1129,37 @@
         "1. Removes the target field from the sampled test data.\n",
         "1. Runs the batch prediction job.\n",
         "1. Evaluates results from the batch prediction job using the ground-truth/target information.\n",
-        "1. Imports the generated evaluation metrics to the Vertex AI model.\n",
-        "\n",
+        "1. Imports the generated evaluation metrics to the Vertex AI model."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9f72ff491712"
+      },
+      "source": [
+        "### Enable the Dataflow API\n",
         "\n",
+        "The model evaluation components from the Google Cloud Pipeline Components create dataflow jobs internally to perform the underlying tasks. So, it is essential to enable the Dataflow API before running the evaluation pipeline."
```

Comment:
Remove "So," and contract "it is":
- It 's essential to enable the Dataflow API before running the evaluation pipeline."